src/dbnode/integration/setup.go

```diff
@@ -160,9 +161,11 @@ func newTestSetup(t *testing.T, opts testOptions, fsOpts fs.Options) (*testSetup
 	// Schema registry is shared between database and admin client.
 	schemaReg := namespace.NewSchemaRegistry(opts.ProtoEncoding(), nil)
 
+	blockLeaseManager := block.NewLeaseManager(nil)
```

Comment:
Hm, won't this cause panics if verifier is nil and add lease is called?

Reply:
No I added guards around that (returns a `verifierIsNotSetErr`)